core/src/main/scala/KubernetesClient.scala

```diff
@@ -173,9 +173,16 @@ object KubernetesJson {
           "spec" := argonaut.Json(
             "containers" := List(
               argonaut.Json(
-                "name"  := stackName.toString,
-                "image" := image.toString,
-                "ports" := containerPortsJson(ports.toList.flatMap(_.nel.list))
+                "name"      := stackName.toString,
+                "image"     := image.toString,
+                "env"       := plan.environment.bindings,
+                "ports"     := containerPortsJson(ports.toList.flatMap(_.nel.list)),
+                "resources" := argonaut.Json(
+                  "requests" := argonaut.Json(
+                    "cpu"    := plan.environment.cpu.getOrElse(0.5),
```

Comment:
Oops need to do this for jobs as well..

Reply:
@timperrett I do wonder, Kubernetes has both a notion of "requests" (lower bound) and "limits" (upper bound). It seems like Nomad's resource stanza is a lower bound with no upper bound? Or is it both. Not having an upper bound seems unsafe.

Reply:
@adelbertc its the other way around - nomad only has upper bounds. Lower bounds don't make sense because then you can't stochastically oversubscribe. So provided you have upper bounds, you should be set.